libsnowflakeclient/include/snowflake_client.h

```diff
@@ -86,6 +86,15 @@ typedef enum sf_error_code {
     SF_ERROR_APPLICATION_ERROR
 } SF_ERROR_CODE;
 
+/**
+ * SQLState for client errors
+ */
+#define SF_SQLSTATE_NO_ERROR "00000"
+#define SF_SQLSTATE_UNABLE_TO_CONNECT "08001"
+#define SF_SQLSTATE_CONNECTION_ALREADY_EXIST "08002"
+#define SF_SQLSTATE_CONNECTION_NOT_EXIST "08003"
+#define SF_SQLSTATE_APP_REJECT_CONNECTION "08004"
```

Comment:
Basically only "08" errors are generated in the client api, while all others come from the server.